client/components/TestQueueRun/index.jsx

```diff
@@ -203,19 +263,44 @@ class TestQueueRow extends Component {
             designPatternLinkOrName = apgExampleName;
         }
 
-        let assignOptions;
-        if (admin) {
-            assignOptions = this.renderAssignDropdowns();
-        } else {
-            assignOptions = this.renderAssignSelfButton(currentUserAssigned);
-        }
+        this.testsCompletedByUser = testers.reduce((acc, uid) => {
+            acc[uid] = testsForRun.filter(
+                t =>
+                    t.results &&
+                    t.results[uid] &&
+                    t.results[uid].status === 'complete'
+            ).length;
+            return acc;
+        }, {});
 
         let testerList = this.renderTesterList(currentUserAssigned);
         let status = 'Not started';
         if (testerList.length) {
             status = 'In Progress';
         }
 
+        let actions;
+        if (admin) {
```

Comment:
Nonblocking: Like in #93, these might be better off as components like `AdminActions` and `TesterActions`